forwarder/internal/logs/models.go

```diff
@@ -126,6 +127,26 @@ func (t AzureLogTime) AsTime() time.Time {
 	return time.Time(t)
 }
 
+type azureLogLevel string
+
+func (l *azureLogLevel) UnmarshalJSON(data []byte) error {
+	var err error
+	var s string
+	err = json.Unmarshal(data, &s)
+	if err == nil {
+		*l = azureLogLevel(s)
+		return nil
+	}
+	// In some cases, log level is an integer.
+	var i int
+	json.Unmarshal(data, &i)
```

Comment:
err = json.Unmarshal(data, &i)
the err below is still the one from the string unmarshal